### PR TITLE
Cleanup IPython bash completion from Jupyter commands.

### PR DESCRIPTION
**Pull request by Carreau**
Requested by @fperez.

### CODE REVIEW

**Comment by fperez** [comment]
Mmh, testing this on a local merge, I'm getting some weird behavior on bash/OSX... If I type `ipython <TAB>`, I get this:

image

Can you replicate this?  Any idea why those spurious 't' letters are appearing?

**Comment by Carreau** [comment]
yep:

```
$ echo '- -' | sed -e 's/ / \t/g'
```

Oh...noooo OS X sed does not respect the gnu standard and does print a `t` instead of a tab but it work on linux.

**Comment by Carreau** [comment]
Sidenote this is present in 4 as well.

**Comment by fperez** [comment]
Yup, it's probably old, I just tested it now.  Remember, OSX userland is BSD, not GNU :)  Lots of little annoying differences like this...

**Comment by fperez** [comment]
BTW, that means we don't necessarily have to fix this as a separate bug. We can merge this and leave the issue open if need be, at least the main ipython/jupyter cleanup got done, and we're no worse than before on this specific issue.

File: examples/IPython Kernel/ipython-completion.bash [hunk 1]
```diff
@@ -24,7 +24,7 @@ _ipython()
 {
     local cur=${COMP_WORDS[COMP_CWORD]}
     local prev=${COMP_WORDS[COMP_CWORD - 1]}
-    local subcommands="notebook qtconsole console kernel profile locate history nbconvert kernelspec install-nbextension trust "
+    local subcommands="kernel profile locate history kernelspec"
```
Comment (minrk) [inline examples/IPython Kernel/ipython-completion.bash]: kernelspec is also a jupyter command, not an IPython one

Comment (Carreau) [inline examples/IPython Kernel/ipython-completion.bash]: > kernelspec is also a jupyter command, not an IPython one

Good call, done.

**Pull request merged**